webpack/ForemanRhCloudTestHelpers.js

```diff
@@ -1,5 +1,6 @@
 export const inventoryStateWrapper = innerState => ({
   ForemanRhCloud: {
     inventoryUpload: { ...innerState },
+    InsightsCloudSync: { ...innerState },
```

Comment:
oh didn't see it, IMO we should create a separate state wrapper or rename the current `inventoryStateWrapper` to `RhCloudStateWrapper`

Reply:
Makes sense, fixed.